org.gnome.Hamster.json

```
@@ -0,0 +1,69 @@
+{
+    "app-id": "org.gnome.Hamster",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "hamster",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--filesystem=xdg-documents",
+        "--own-name=org.gnome.Hamster",
```

```suggestion
        "--own-name=org.gnome.Hamster"
```
This should fix the build.